### PR TITLE
frontend/feat/faq-yatri-coins

### PR DESCRIPTION
**Pull request by yashikakaushik**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-driver/src/Screens/DriverEarningsScreen/ComponentConfig.purs [hunk 1]
```diff
@@ -57,7 +57,10 @@ genericHeaderConfig state =
     height = WRAP_CONTENT
     , padding = if state.props.subView == ST.EARNINGS_VIEW then Padding 16 16 16 16 else PaddingVertical 5 5
     , textConfig {
-        text = if not state.data.config.coinsConfig.enableYatriCoins then getString EARNINGS else getString USE_COINS
+        text = if state.props.subView == ST.FAQ_VIEW then "Yatri Coins FAQs"
```
Comment (DwivediAman-exe) [inline Frontend/ui-driver/src/Screens/DriverEarningsScreen/ComponentConfig.purs:60]: @yashikakaushik add translations for this.

File: Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs [hunk 2]
```diff
@@ -661,6 +686,244 @@ dummyBages = [{badgeImage: "ny_ic_five_star_badge,https://assets.juspay.in/namma
...
+                    , margin $ MarginRight 16
+                    ] <> FontStyle.subHeading2 TypoGraphy)
+                ]
+              , linearLayout
+                [ height MATCH_PARENT
+                , width $ V 20
+                , orientation HORIZONTAL
+                , gravity CENTER
+                  ][ imageView
+                      [ imageWithFallback $ "ny_ic_chevron_right_grey,"
```
Comment (DwivediAman-exe) [inline Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs:795]: fetchImage FF_ASSET <image_name>

File: Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs [hunk 3]
```diff
@@ -661,6 +686,244 @@ dummyBages = [{badgeImage: "ny_ic_five_star_badge,https://assets.juspay.in/namma
...
+                      , height $ V 18
+                      , width $ V 18
+                      , margin $ Margin 10 0 8 0
```
Comment (DwivediAman-exe) [inline Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs:798]: use marginhorizontal

File: Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs [hunk 4]
```diff
@@ -661,6 +686,244 @@ dummyBages = [{badgeImage: "ny_ic_five_star_badge,https://assets.juspay.in/namma
...
+  linearLayout
+    [ height MATCH_PARENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    ] $ [faqVid push state] <> if length answersList > 1 
+        then (map (\item -> 
+          linearLayout 
+            [ height MATCH_PARENT
+            , width MATCH_PARENT
+            , orientation HORIZONTAL
```
Comment (DwivediAman-exe) [inline Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs:863]: We can remove orientation HORIZONTAL everywhere as it is default

File: Frontend/android-native/app/src/driver/yatriPartner/assets/juspay/juspay_assets.json [hunk 5]
```diff
@@ -72,6 +73,7 @@
       "ic_graph_black" : true,
       "ic_grey_border" : true,
       "ic_hatchback" : true,
+      "ic_help_circle_blue" : true,
```
Comment (DwivediAman-exe) [inline Frontend/android-native/app/src/driver/yatriPartner/assets/juspay/juspay_assets.json:76]: naming format should be ny_ic....

**Review by DwivediAman-exe — COMMENTED** [review]
Also compress the image sizes

File: Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs [hunk 6]
```diff
@@ -661,6 +686,235 @@ dummyBages = [{badgeImage: "ny_ic_five_star_badge,https://assets.juspay.in/namma
...
+    ] $ [faqVid push state] <> if length answersList > 1 
+        then (map (\item -> 
+          linearLayout 
+            [ height MATCH_PARENT
+            , width MATCH_PARENT
+            , padding (PaddingVertical 10 10)
+            ][ textView (
+                [ height WRAP_CONTENT
+                , width WRAP_CONTENT
+                , text "•"
```
Comment (MercyQueen) [inline Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs:862]: this text can be appended to the next one.This way we can remove one textView

File: Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs [hunk 7]
```diff
@@ -661,6 +686,235 @@ dummyBages = [{badgeImage: "ny_ic_five_star_badge,https://assets.juspay.in/namma
...
+        , height (V 22)
+        , imageWithFallback $ fetchImage FF_ASSET "ny_ic_help_circle_blue,"
+        , padding $ Padding 0 2 4 0 
+        , gravity CENTER_VERTICAL
+        , onClick push $ const $ ChangeTab ST.FAQ_VIEW
+        ]
+      , textView $ 
+        [ height WRAP_CONTENT
+        , width WRAP_CONTENT
+        , text "Help"
```
Comment (MercyQueen) [inline Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs:912]: add translations

File: Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs [hunk 8]
```diff
@@ -661,6 +686,235 @@ dummyBages = [{badgeImage: "ny_ic_five_star_badge,https://assets.juspay.in/namma
...
+answersListView :: forall w . (Action -> Effect Unit) -> Array String -> ST.DriverEarningsScreenState -> PrestoDOM (Effect Unit) w
+answersListView push answersList state = 
+  linearLayout
+    [ height MATCH_PARENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    ] $ [faqVid push state] <> if length answersList > 1 
+        then (map (\item -> 
+          linearLayout 
+            [ height MATCH_PARENT
```
Comment (MercyQueen) [inline Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs:856]: create a function for this linearLayout and also for the one inside else.

File: Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs (hunk 6 above)
Comment (MercyQueen) [reply Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs:862]: one single map function can be used.

**Review by MercyQueen — CHANGES_REQUESTED** [review]

**Pull request merged**